tests/test_transformDiaSourceCatalog.py

```diff
@@ -102,12 +106,36 @@ def test_run(self):
         np.testing.assert_array_equal(result.diaSourceTable["midPointTai"],
                                       [self.date.get(system=dafBase.DateTime.MJD)]*self.nSources)
         np.testing.assert_array_equal(result.diaSourceTable["diaObjectId"], [0]*self.nSources)
+        np.testing.assert_array_equal(result.diaSourceTable["x"], np.arange(self.nSources))
```

Comment:
I don't understand what the new `x` and `y` fields have to do with testing sky sources?

Reply:
I wanted another way to distinguish that the source ordering was correct: I did this ticket because in my last ticket on this package I got the order wrong in my first attempt, and this test would have caught that.